LICENSE

```diff
@@ -1,21 +1,32 @@
-MIT License
+GNU GENERAL PUBLIC LICENSE
```

Comment:
The original request to open source this package recommended BSD-3 clause, so that would be better.